### PR TITLE
Import: Dockerfile edge case

### PR DESCRIPTION
**Pull request by pyrmont**
This imports jneen#1070.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ cache:
   bundler: true
 bundler_args: --without development
 before_install:
-  - gem install bundler
+  - gem install bundler -v 1.17.2
```
Comment (pyrmont) [inline .travis.yml]: This branch includes updated Travis settings and this line is no longer required.

Comment (pyrmont) [inline .travis.yml]: This branch includes updated Travis settings and this line is no longer required.

**Review by pyrmont — COMMENTED** [review]
@sethbergman When you have a chance, could you fix this?

File: .travis.yml (hunk 1 above)
Comment (pyrmont) [reply .travis.yml]: Thanks!

Comment (sethbergman) [reply .travis.yml]: No problem!

**Pull request merged**